src/main/java/org/broadinstitute/hellbender/tools/exome/GetHetCoverage.java

```diff
@@ -98,16 +99,19 @@
     )
     protected int minimumBaseQuality = 20;
 
+    @Argument(
+            doc = "Validation stringency for all BAM files read by this program.  Setting stringency to SILENT " +
+            "can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) " +
+            "do not otherwise need to be decoded.",
+            common=true)
+    protected ValidationStringency VALIDATION_STRINGENCY = ValidationStringency.DEFAULT_STRINGENCY;
```

Comment:
thanks for pointing this out.  i had actually originally implemented something more along the lines of what you suggested below, but then changed it after seeing how things were done in PicardCommandLineProgram---didn't appreciate that things were being done differently for the gatk tools!